docs/OnceFrom.md

```diff
@@ -50,5 +50,5 @@ Where `example.ign` here is the content from https://github.com/coreos/ignition/
 ```
 
 You can also try out the MachineConfig support of "once-from" mode by passing a
-MC manifest instead. See the contents of HACKING.md for an example MachineConfig
+MC manifest instead. See the contents of [HACKING.md](./HACKING.md) for an example MachineConfig
```

Comment:
Since we are changing this, can we simplify this entire sentence to : "See [HACKING.md](./HACKING.md) for an example MachineConfig."

Reply:
ack